File: js/class-3/four.js
let a={} //js-object
let emp={
    "eid":101,
    "ename":"madhu"
    }
//read object
console.log(a)
console.log(emp)
console.log(emp.eid)
console.log(emp.ename)
console.log(emp.loc)
//update
emp.esal="5000.55"
console.log(emp)//add
emp.eid=102
console.log(emp)
console.log(typeof(a))
//delete
delete emp.eid
console(emp)

File: PYTHON/requestmodule/four.py
import csv
fp=open('emp.csv','w',newline='')
cw=csv.writer(fp)
cw.writerow(["eid","ename","esal"])
data=[(101,"rg",45000),
      (102,"sg",35000),
      (103,"pg",45000)
    ]
cw.writerows(data)
print("new file created")
fp.close()
